packages/@aws-cdk-testing/framework-integ/test/aws-apigateway/test/integ.restapi.import-deploymentstage.ts

```diff
@@ -0,0 +1,23 @@
+import * as cdk from 'aws-cdk-lib/core';
+import * as apigateway from 'aws-cdk-lib/aws-apigateway';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'integtest-restapi-import-deployment-stage');
+
+const api = new apigateway.RestApi(stack, 'my-api', {
+  deploy: false,
```

Comment:
if `deploy:false` is used, seems we need to use [deploymentStage](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_apigateway.RestApi.html#deploymentstage) to point the stage explicitly to the restapi.